src/App.css

```diff
@@ -0,0 +1,23 @@
+body {
+  background-image: url('assets/images/bg.jpg');
```

Comment:
- [ ] Whe mobile view is questionable because some content won't fit the screen to be viewed properly:

![image](https://github.com/BHObserver/Rent-a-Car-frontend/assets/57421823/9ce34bb8-3082-49ba-b0df-a61463d22425)

Kindly, make it so the content fits. 👍